src/box/lua/tuple.c

```diff
@@ -779,12 +777,17 @@ box_lua_tuple_init(struct lua_State *L)
 	luaL_findtable(L, LUA_GLOBALSINDEX, "box.internal", 1);
 	luaL_newmetatable(L, tuplelib_name);
 	luaL_setfuncs(L, lbox_tuple_meta, 0);
+	lua_pushcfunction(L, lbox_tuple_new);
+	lua_setfield(L, -2, "new");
```

Comment:
Nit: Can't you simply add this function to `lbox_tuple_meta` instead?

Reply:
Fixed.